lib/gui/app/components/wifi/react/wifi.jsx

```diff
@@ -453,8 +453,9 @@ class Wifi extends React.PureComponent {
    */
   handleError (err) {
     console.error(err)
+    const message = (!_.isEmpty(err.data) && _.head(err.data)) || err.message
     this.setState({
-      wifiSettingsError: err
+      wifiSettingsError: { message }
```

Comment:
`new Error( message )` please ;P

Reply:
Yeah, and I should definitely keep this in mind